lib/qa/authorities/linked_data/search_query.rb

```diff
@@ -15,8 +15,8 @@ def initialize(search_config)
         @search_config = search_config
       end
 
-      attr_reader :search_config, :full_graph, :filtered_graph, :language, :access_time_s, :normalize_time_s, :fetched_size, :normalized_size, :subauthority
-      private :full_graph, :filtered_graph, :language, :access_time_s, :normalize_time_s, :fetched_size, :normalized_size, :subauthority
+      attr_reader :search_config, :full_graph, :filtered_graph, :language, :access_time_s, :normalize_time_s, :subauthority, :request_header
+      private :full_graph, :filtered_graph, :language, :access_time_s, :normalize_time_s, :subauthority, :request_header
```

Comment:
* Make request_header available throughout.  
* Calculations for fetched_size and normalized size moved to method generating the performance data to return with the results.